templates/cairo0/protostar.toml

```diff
@@ -1,3 +1,7 @@
-["protostar.project"]
+[project]
+protostar-version = "0.0.0"
+lib-path = "lib"
+
+[contracts]
+main = ["src/main.cairo"]
 
-["protostar.contracts"]
```

Comment:
protostar.toml in templates has no impact on how it is created during `init` command call, but it has to be this way to ensure it is the same as the one we create [here](https://github.com/software-mansion/protostar/blob/816cba84923c2d002e677ebc14289b82777682af/protostar/commands/init/project_creator/_project_creator.py#L28) for tests